README.md

```diff
@@ -216,8 +232,38 @@ go get ./...
 go build
 ```
 
+## Event Filtering
+
+The 4.X event forwarder supports custom filter defintions via golang's templating language to filter the stream of Cb Event Messages coming over the event-forwarder.
+
+```
+filter:
+    enabled: true
+    template: >-
+              {{if (eq .type "alert.watchlist.hit.query.binary") -}}
+                KEEP
+              {{- else -}}
+                DROP
+              {{- end}}
+```
+
+##Encoders / Formating Events with Templates
+
+The 4.X event forwarder supports custom event formatting defintions using golang's templating language to format the stream of Cb Event Messages coming over the event-forwarder based on template configured by the user.
+
+Below is an example using the yaml syntax >- to open a whitespace clipped multi-line string sans-quotes. 
+
+```
+output_format: template
+encoder:
+    template: >- 
+              {{$cbmsg := . }}
+              {{- JsonFormat $cbmsg -}  
```

Comment:
Two things here:

1. We need a list of the available functions/filters (`JsonFormat`, etc.) For each one, we need an example of the output that's generated, any options accepted for that function, and a short description of why/how to use it
2. Let's give a more concrete example - for example with the Sumologic integration example that we were given.